src/FilecoinPayV1.sol

```diff
@@ -1323,25 +1323,21 @@ contract FilecoinPayV1 is ReentrancyGuard {
     {
         Rail storage rail = rails[railId];
         RateChangeQueue.Queue storage rateQueue = rail.rateChangeQueue;
+        Account storage payee = accounts[rail.token][rail.to];
```

Comment:
I think the code gets a little simpler if you calculate the `payee` in `settleRailInternal` too